apps/vx-scan/frontend/src/app.test.tsx

```diff
@@ -93,7 +79,7 @@ beforeEach(() => {
   );
   window.kiosk = kiosk;
 
-  apiMock.mockApiClient.reset();
+  apiMock = createApiMock();
```

Comment:
The `reset` command didn't quite suit my needs since I've overridden some of the default mocks with jest mocks. Instead of reasoning through how to cleanly reset everything in a `reset` wrapper, I found it easier to just create a fresh mock for every test

To avoid gotchas down the road in other VxScan tests, I may proactively update all VxScan tests to take this approach

Reply:
Go for it. I saw that you used this pattern in the other apps, so I think making it consistent makes sense. Feel free to remove the `reset` method from the mock API client completely and make this the standard approach.